recipes/net/iproute2.yaml

```diff
@@ -0,0 +1,33 @@
+inherit: [cpackage, make, install]
+
+metaEnvironment:
+    PKG_VERSION: "6.13.0"
+    PKG_LICENSE: "GPL-2.0+"
+
+checkoutSCM:
+    scm: url
+    url: https://mirrors.edge.kernel.org/pub/linux/utils/net/iproute2/iproute2-${PKG_VERSION}.tar.xz
```

Comment:
There's the `KERNEL_MIRROR` variable from `basement` -> `${KERNEL_MIRROR}/linux/utils/net/iproute2/iproute2-${PKG_VERSION}.tar.xz`.